app/views/posts/show.html.erb

```diff
@@ -187,7 +187,15 @@
 <% if @post.spam_domains.any? %>
   <ul class="post-domain-list">
     <% @post.spam_domains.each do |d| %>
-      <li><%= render 'spam_domains/domain', domain: d, occurances: true %></li>
+      <li>
+        <%= render 'spam_domains/domain', domain: d, occurances: true %>
+        <% if current_user&.has_role?(:core) && !(psd = PostSpamDomain.find_by(post: @post, spam_domain: d)).added_by.nil? %>
```

Comment:
Declaring variables inside conditional statements is... not wonderful. The declaration should move to the line above and just be used in the conditional.

Reply:
I actually really like this style :( I'm pretty sure rubocop is OK with it as well.

Reply:
Not fixing for now because I want to get this in before the deploy, but let me know if you feel strongly about it and we can fix it on master.